config/service_alias.go

```diff
@@ -22,9 +22,49 @@ const (
 // gateway.QoSService interface.
 // THis is to handle requests for a given service ID.
 var ServiceQoSTypes = map[protocol.ServiceID]ServiceQoSType{
-	// TODO_IMPROVE(@commoddity): Add all EVM service IDs here.
+	// All Morse EVM Services as of 12/17/2024
```

Comment:
We should move these into a separate map called `legacyMorseMapping` and have the main map be an aggregation of the two.

Reply:
Is this a requirement now or a nice to have that we can sideline for later? This is all going to be ripped out and replaced per #104 with Envoy Proxy